driver/src/metrics/stablex_metrics.rs

```diff
@@ -106,29 +106,27 @@ impl StableXMetrics {
         }
     }
 
-    pub fn auction_solution_computed(&self, batch: U256, orders: &[Order], res: &Result<Solution>) {
+    pub fn auction_solution_computed(&self, batch: U256, res: &Result<Solution>) {
         let stage_label = &[ProcessingStage::Solved.as_ref()];
         let book_label = &[BookType::Solution.as_ref()];
         self.processing_times
             .with_label_values(stage_label)
             .set(time_elapsed_since_batch_start(batch));
         match res {
             Ok(solution) => {
-                let touched_orders = orders
-                    .iter()
-                    .zip(&solution.executed_buy_amounts)
-                    .filter(|(_, &amount)| amount > 0u128)
-                    .map(|(o, _)| o.clone())
-                    .collect::<Vec<Order>>();
-                self.orders
-                    .with_label_values(book_label)
-                    .set(touched_orders.len().try_into().unwrap_or(std::i64::MAX));
+                self.orders.with_label_values(book_label).set(
+                    solution
+                        .executed_orders
+                        .len()
+                        .try_into()
+                        .unwrap_or(std::i64::MAX),
+                );
                 self.tokens
                     .with_label_values(book_label)
-                    .set(tokens_from_orders(&touched_orders));
+                    .set(tokens_from_solution(solution));
```

Comment:
@fleupold This does the same thing as before, right? (see the implementation of this function below)

Reply:
Is OWL included in the price vector? I know that it is not passed on the smart contract (as it's always assumed to be 10**18). But it should be considered a token for logging sake.

Reply:
Ok, it's filtered out here, so we should be fine: https://github.com/gnosis/dex-services/blob/4e7548e373d6c8909063229ea771d1d791c42fe2/driver/src/contracts/stablex_contract.rs#L182